lib/rules/selector-combinator-space-after/__tests__/index.js

```diff
@@ -135,79 +136,136 @@ testRule(rule, {
     {
       code: "namespace|type#id > .foo {}",
       description: "qualified id with namespace"
+    },
+    {
+      code: ", a {}",
+      description: "A selector that starts with a comma"
+    },
+    {
+      code: ", /*c*/a {}",
+      description: "A selector that starts with a comma & include comment"
     }
   ],
 
   reject: [
     {
       code: "a+  a {}",
+      fixed: "a+ a {}",
       description: "two spaces after + combinator",
       message: messages.expectedAfter("+"),
       line: 1,
       column: 2
     },
     {
       code: "a+\na {}",
+      fixed: "a+ a {}",
       description: "newline after + combinator",
       message: messages.expectedAfter("+"),
       line: 1,
       column: 2
     },
     {
       code: "a+a {}",
+      fixed: "a+ a {}",
       description: "no space after + combinator",
       message: messages.expectedAfter("+"),
       line: 1,
       column: 2
     },
     {
       code: "a>a {}",
+      fixed: "a> a {}",
       description: "no space after > combinator",
       message: messages.expectedAfter(">"),
       line: 1,
       column: 2
     },
     {
       code: "a~a {}",
+      fixed: "a~ a {}",
       description: "no space after ~ combinator",
       message: messages.expectedAfter("~"),
       line: 1,
       column: 2
     },
     {
       code: "a + .foo.bar ~a {}",
+      fixed: "a + .foo.bar ~ a {}",
       description: "multiple combinators: no space after ~ combinator",
       message: messages.expectedAfter("~"),
       line: 1,
       column: 14
     },
     {
       code: "#foo +.foo.bar ~ a {}",
+      fixed: "#foo + .foo.bar ~ a {}",
       description: "multiple combinators: no space after + combinator",
       message: messages.expectedAfter("+"),
       line: 1,
       column: 6
     },
     {
       code: "a >>>a {}",
+      fixed: "a >>> a {}",
       description: "shadow-piercing descendant combinator",
       message: messages.expectedAfter(">>>"),
       line: 1,
       column: 3
     },
     {
       code: "namespace|type#id >.foo {}",
+      fixed: "namespace|type#id > .foo {}",
       description: "qualified id with namespace",
       message: messages.expectedAfter(">"),
       line: 1,
       column: 19
+    },
+    {
+      code: "a >a >a {}",
+      fixed: "a > a > a {}",
+      message: messages.expectedAfter(">"),
+      line: 1,
+      column: 3
+    },
+    {
+      code: "a/*comment*/>/*comment*/a {}",
+      fixed: "a/*comment*/> /*comment*/a {}",
+      description: "comment",
+      message: messages.expectedAfter(">"),
+      line: 1,
+      column: 13
+    },
+    {
+      code: "a/*comment*/>/*comment*/a, b/*comment*/>/*comment*/b {}",
+      fixed: "a/*comment*/> /*comment*/a, b/*comment*/> /*comment*/b {}",
+      description: "comment",
+      message: messages.expectedAfter(">"),
+      line: 1,
+      column: 13
+    },
+    {
```

Comment:
Can you move this and the following tests into the `syntax: scss` `testRule` block at the bottom of the file as nesting without the _nesting selector_ (`&`) isn't in the draft spec?